app/decorators/reviewing_asset.rb

```diff
@@ -5,6 +5,11 @@ def review!
     save
   end
 
+  def unreview!
```

Comment:
I don't think it is being called anywhere. I was thinking this could still be useful if we want to unreview something from say the item record page or the search results, but that may be a discussion for later. For now I can definitely remove it.